webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var path = require('path')
+// var nodeExternals = require('webpack-node-externals')
+
+module.exports = {
+  mode: process.env.NODE_ENV,
+  entry: './hyper-readings.es.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'hyper-readings.js',
+    library: 'HyperReadings',
+    libraryTarget: 'umd',
+    umdNamedDefine: true,
+    globalObject: `typeof self !== 'undefined' ? self : this`
+  },
+  externals: [
+    'fs',
+    'crypto'
```

Comment:
when bundling in hypercore we need to explicitly let it know that crypto is available as an external library, otherwise it will throw errors.